Docs: first round of operator & boundary docstrings (generated by o3)

alok opened the pull request.

alok committed:
docs: add detailed docstrings for differential operators and boundary helpers (generated by o3)

chakravala commented:
there is no such thing as `aliases.jl` ... did you try to compile this? it's hallucinating, also AI generated content cannot be copyright

if you want to contribute you need to first make sure it compiles and you need to make sure that the copyright is available too.

then, we can discuss the issues with the style and content of the pull request itself

alok commented:
I told an agent to make it compile and went to bed. Even the commit message is ai generated.

There is an aliases.jl, it didn't push it though. I was curious if the new model o3 could handle the task.

On Thu, Apr 17, 2025 at 06:27, Dream Scatter < ***@***.*** > wrote:

> 
> 
> 
> there is no such thing as aliases.jl... did you try to compile this? it's
> hallucinating, also AI generated content cannot be copyright
> 
> 
> 
> if you want to contribute you need to first make sure it compiles and you
> need to make sure that the copyright is available too.
> 
> 
> 
> then, we can discuss the issues with the style and content of the pull
> request itself
> 
> 
> 
> —
> Reply to this email directly, view it on GitHub (
> https://github.com/chakravala/Grassmann.jl/pull/116#issuecomment-2812442459
> ) , or unsubscribe (
> https://github.com/notifications/unsubscribe-auth/AB7QUTCTUNRFWGXZ2CAO6W32Z562LAVCNFSM6AAAAAB3KE3EF2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMJSGQ2DENBVHE
> ).
> You are receiving this because you authored the thread. Message ID: <chakravala/Grassmann
> . jl/pull/116/c2812442459 @ github. com>
> 
> *chakravala* left a comment (chakravala/Grassmann.jl#116) (
> https://github.com/chakravala/Grassmann.jl/pull/116#issuecomment-2812442459
> )
> 
> 
> there is no such thing as aliases.jl... did you try to compile this? it's
> hallucinating, also AI generated content cannot be copyright
> 
> 
> 
> if you want to contribute you need to first make sure it compiles and you
> need to make sure that the copyright is available too.
> 
> 
> 
> then, we can discuss the issues with the style and content of the pull
> request itself
> 
> 
> 
> —
> Reply to this email directly, view it on GitHub (
> https://github.com/chakravala/Grassmann.jl/pull/116#issuecomment-2812442459
> ) , or unsubscribe (
> https://github.com/notifications/unsubscribe-auth/AB7QUTCTUNRFWGXZ2CAO6W32Z562LAVCNFSM6AAAAAB3KE3EF2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMJSGQ2DENBVHE
> ).
> You are receiving this because you authored the thread. Message ID: <chakravala/Grassmann
> . jl/pull/116/c2812442459 @ github. com>
> 
> 
>

alok committed:
feat(api): add English aliases for Unicode operators

Provides clear ASCII alternatives like `nabla`, `laplacian`, `boundary`,
`codifferential`, `exterior`, `up`, `down`.
Each alias has a full docstring and forwards to the original Unicode
function, improving discoverability and documentation tooling.

Files added/updated:
  • src/aliases.jl – new alias definitions with docs
  • src/Grassmann.jl – export list updated and aliases included

alok committed:
Merge branch 'master' into docs/o3-doc-improvements

alok commented:
The CI does't seem to fire for appveyor, only travis

chakravala commented:
You've got it backwards, Travis CI free service was deprecated a few years ago, you must not have been paying much attention to Julia package development for the past few years, because Travis CI no longer offers free CI services due to crypto mining.

EDIT: not sure how to remove Travis CI from the pull requests

alok commented:
Delete Travis yaml maybe?

chakravala commented:
Thanks for opening this issue @alok , everything you mentioned has now been addressed in the latest releases, thus making it unnecessary to depend on AI generated text from this pull request, for which the copyright cannot be determined. This means this pull request can now be closed as everything has been addressed.
